go/client/provision_ui.go

```diff
@@ -106,7 +106,7 @@ func (p ProvisionUI) DisplayAndPromptSecret(ctx context.Context, arg keybase1.Di
 		p.parent.Output("It will then prompt you for the verification code above.\n\n")
 
 		if arg.OtherDeviceType == keybase1.DeviceType_MOBILE {
-			encodings, err := qrcode.Encode(arg.Secret)
```

Comment:
We're encoding the phrases and not the secret. the barcode scanning code on ios needs the qr output to be phrases and not binary for now